bundlegen/cli/main.py

```diff
@@ -157,6 +157,9 @@ def generate(image, outputdir, platform, searchpath, creds, ipk, appmetadata, ye
         shutil.rmtree(outputdir)
         sys.exit(2)
 
+    # Validate the app metadata & platform config files against the reference schemas
+    processor.validateWithSchema()
```

Comment:
Since this throws an exception, wrap in a try/except so we can print a nice error.

Either that, or change `validateWithScema` to return a bool